src/LinuxTracing/ThreadStateManager.cpp

```diff
@@ -78,7 +91,13 @@ std::optional<ThreadStateSlice> ThreadStateManager::OnSchedWakeup(uint64_t times
   slice.set_thread_state(open_state.state);
   slice.set_duration_ns(timestamp_ns - open_state.begin_timestamp_ns);
   slice.set_end_timestamp_ns(timestamp_ns);
-  tid_open_states_.insert_or_assign(tid, OpenState{kNewState, timestamp_ns});
+  slice.set_wakeup_tid(open_state.wakeup_tid);
+  slice.set_wakeup_pid(open_state.wakeup_pid);
+  slice.set_wakeup_reason(open_state.wakeup_reason);
+  tid_open_states_.insert_or_assign(
+      tid, OpenState{kNewState, timestamp_ns, was_blocked_by_tid, was_blocked_by_pid,
+                     orbit_grpc_protos::ThreadStateSlice::WakeupReason::
+                         ThreadStateSlice_WakeupReason_kUnblocked});
```

Comment:
`orbit_grpc_protos::ThreadStateSlice::WakeupReason::kUnblocked` or `orbit_grpc_protos::ThreadStateSlice::kUnblocked` (not sure which one works)